### DISCUSSION TITLE
Minor Navigation Bar Improvement

### DISCUSSION THREAD

**Discussion by ernst86913** [directus/directus#16908] [Feature Requests]
Hi - 
currently the logo and the avatar are reloaded on every route change... which creates a bit of a "messy" experience for the user as those components are "flickering". Would it be possible to keep those 2 components stay on the screen like all other navigation bar components (e.g. the Directus Standard logo works nicely).

Thanks,
Roman

**Comment by rijkvanzanten** [comment]
Odd! Lets track this as a bug, cause it shouldn't be doing that (I'm assuming it has something to do with the fact that it's loaded from the API and injected as a data string, not sure)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
